04-ruby-mvc-sinatra/ruby-basics/readme.md

```diff
@@ -34,19 +34,19 @@ Ruby resides in and is processed by a webpage's server.
 * That means you **cannot** look at or mess with a site's Ruby code as you did with Javascript.
 * We won't be including Ruby files in our HTML documents. You'll learn more about back-end development and how to connect everything together next week.
 
-#### What's Ruby like? (5min)
+#### What's Ruby like? (4min)
 
 MINASWAN
 * Does anybody remember this from Allie's lunchtime presentation last Friday?
   * "Matz Is Nice And So We Are Nice"
     * Mentality not only applies to how you should treat your fellow developers, but also the philosophy behind Ruby itself
   * Yukihiro Matsumoto ("Matz") created Ruby to increase developer happiness.
+  ![He really is quite nice.](matz-n-park.png)
```

Comment:
I think we should make a point of saying while this is awesome lets not all got tweet at matz now